README.md

```diff
@@ -55,11 +55,10 @@ Your guests will configure themselves.
 
 Be sure to start and run the environment with `make start` before you
 continue with these steps. You must have working vagrant, virtualbox, and
-ansible.
+ansible. If you are behind a proxy server, then use only http proxy.
```

Comment:
I'm not sure what this means, can you clarify it? Maybe reference `http_proxy` and `https_proxy` as ways to configure the makefile.